### PR TITLE
New package: pamixer

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

**Comment by Vaelatern** [comment]
The rule within Void is that for every template altered, there should be one commit. Those commits should be named as directed by the end of the [Manual (link)](https://github.com/voidlinux/void-packages/blob/master/Manual.md#contributing-via-git).

**Only one PR, please, unless you are told otherwise. Unrelated features, such as if you fixed something in `firefox` and added a new spotify client, should go in separate PRs. If you wanted to package something, and it depended on a different package not yet in Void, you would put both of those package additions in the *same* PR. Please do not close and open new PRs just because you made changes. Learn enough git commands, specifically 'rebase' and 'push' with the '--force' flag to not need to do this**.

**Comment by teldra** [comment]
Sry, somthing br0ke horrendous.

**Comment by Vaelatern** [comment]
Okay. Rebase to name the commit correctly, then force push? I think a `commit --amend` would do to rename it actually, to match the manual.
typically a rebase on top of the master branch will fix most ills. Rebase a few times and most everything fixes itself.

**Comment by Vaelatern** [comment]
Message of the commit, not name of the issue, needs to conform.

**Comment by teldra** [comment]
Is this correct?

**Comment by Vaelatern** [comment]
Yes

File: srcpkgs/pamixer/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'pamixer'
+pkgname=pamixer
+version=1.3.1
+revision=1
+build_style=gnu-makefile
+makedepends="pulseaudio-devel boost-devel"
+short_desc="Pulseaudio command line mixer "
```
Comment (Vaelatern) [inline srcpkgs/pamixer/template]: Please don't end in whitespace either.

**Pull request merged**